etc/kayobe/inventory/group_vars/all/dev-tools

```diff
@@ -0,0 +1,8 @@
+---
+########################################################################
+# List of development tool packages to install during host configuration
+dev_tools_packages_extra:
+  - tcpdump
+  - jq
+  - wget
+  - htop
```

Comment:
Maybe https://github.com/aristocratos/btop as an alternative.

Reply:
Similarly to `htop`, this is only in EPEL for RL9.

Reply:
Added, but made it conditional